components/automate-ui/src/app/page-components/user-table/user-table.component.scss

```diff
@@ -11,3 +13,13 @@ chef-table {
     justify-content: flex-end;
   }
 }
+
+.team-users-empty-state {
```

Comment:
we'd like to make this `empty-state` rule shareable across components, but waiting on @chef/ui-team for some guidance. just adding it to `ui.component.scss` didn't cut it

Reply:
seems like the best move would be to create an empty state component with the styles we want, so I'll make a card for that this week